recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -580,12 +567,11 @@ def validate(self):
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
-        self._patch_sources()
+        apply_conandata_patches(self)
 
     def _enabled_sdks(self):
-        for sdk in self._sdks:
-            if self.options.get_safe(sdk[0]):
-                yield sdk
+        return [sdk for sdk in self._sdks
+                if self.options.get_safe(sdk)]
```

Comment:
The change here is that sdk is no longer a tuple, so we need to remove the subscript access